Data/Sys/GameSettings/SZA.ini

```diff
@@ -6,7 +6,7 @@
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
 EmulationStateId = 4
-EmulationIssues = Config and use the quitar and drums, wiimote asks for a microphone otherwise.(r6575)
+EmulationIssues = Config and use the quitar and drums, wiimote asks for a microphone otherwise.
```

Comment:
Hmm, it was meant to say that without configuring the guitar and drums controls and using them, with only a wiimote connected and configured, it will ask for a microphone instead and you won't be able to progress ingame.